AstarteDeviceSDKCSharp/AstartePairingService.cs

```diff
@@ -162,7 +161,8 @@ public AstartePairingService(string pairingUrl, string astarteRealm)
                               "Request to Pairing API failed with "
                                   + response.StatusCode.ToString()
                                   + ". Returned body is "
-                                  + response.Content.ToString());
+                                  + (response.Content != null ?
+                                  await response.Content.ReadAsStringAsync() : "empty"));
```

Comment:
same above

Reply:
Since the `response.Content` is not nullable I made a new solution.